NSS-Likeable-Lemurs/BangazonSite
# Hc search bar ticket6

## Link to Ticket
Closes # [Link](https://github.com/NSS-Likeable-Lemurs/BangazonSite/issues/6)

## Description of Proposed Changes
- Added search bar with functionality. 
- Search bar only visible when logged in

## Steps to Test

```sh
git fetch --all
git checkout <HC-SearchBar-Ticket6>
1. Log into the account and click in search bar.
2. Put in a Product name.
3. Press enter.
4.  See products pop up
```

## Impacted Areas in Application

List general components of the application that this PR will affect:

• _Login.Partial
• Products controller
• SearchResults.cshtml
## Mentions @kellycook301 @ThraxHamilton @streatorward

## Definition of Done

1. The project must be fully documented. This includes the following:
    1. Complete README that documents the steps to install the code, how to install any dependencies, or system configuration needed.
    1. Every class must be documented with purpose, author, and methods.
    1. Every method must be documented with purpose and argument list - which itself must contain a short purpose for each argument.
1. The project must be able to run fully, and without errors, on each teammate's system.
1. Fulfills every requirement.
Every line of code has been peer reviewed.
1. For projects that require unit testing, core functionality must be identified and have at least one test for each.
